_data/sites.json

```diff
@@ -10770,6 +10770,16 @@
         ]
     },
 
+    {
+        "name": "Litmus",
+        "url": "https://help.litmus.com/article/420-delete-your-account",
+        "difficulty": "limited",
+        "notes": "They offer a \"simple deletion\" where your account will be dissociated with your email address, and a [\"full data deletion\"](https://trust.litmus.com/?itemUid=af8eb7bc-d4b7-4e13-a4b7-6f05e2744c22) for complete data deletion.",
```

Comment:
On the full data deletion do they verify that you're in a country with "deletion rights"?

Reply:
Nope, didn't have to provide anything.
Maybe they know based on IP from form submission or account usage 🤷

Reply:
Likely if you choose a wrong country they don't check, but since we don't know, I'll keep it as `limited`